src/pip/_internal/commands/install.py

```diff
@@ -327,34 +364,18 @@ def run(self, options, args):
                         else:
                             legacy_requirements.append(req)
 
-                    # We don't build wheels for legacy requirements if we
-                    # don't have wheel installed or we don't have a cache dir
-                    try:
-                        import wheel  # noqa: F401
-                        build_legacy = bool(options.cache_dir)
```

Comment:
Could you make a commit changing/replacing this with `build_legacy = True` followed by the cleanup?

Reply:
Sure. (The cleanup was more to add a test of the code rather than cleanup per se.) Should I let the tests run after doing that commit, or was it more just for the history?

Reply:
Alternatively, I could do the refactoring first (with the tests), followed by the behavior change.

Reply:
The alternative sounds best. I care more about the history / review-ability than the test suite being operational throughout. ;)